app/presenters/concerns/application_transitioning_module.rb

```diff
@@ -0,0 +1,10 @@
+# to remove once Notes and Withdrawals add the required fields
+module ApplicationTransitioningModule
```

Comment:
I don't think we should be introducing this at this stage, I think we can wait till the API version is released, then we can start validating against the schema, something to discuss today in the API meeting